solidity/contracts/TokenStaking.sol

```diff
@@ -589,7 +589,13 @@ contract TokenStaking is StakeDelegatable {
 
         bool isActive = _isInitialized(operatorParams);
 
-        if (isAuthorized && isActive) {
+        bool stakeReleased = _isStakeReleased(
```

Comment:
We should update the documentation to this function. Right now it says nothing about the fact the stake can't be released. We also need to update the documentation in `KeepRandomBeaconOperator.sol` for `hasMinimumStake` function.

Reply:
The documentation for `hasMinimumStake` in the operator contract doesn't really need updating because it doesn't refer to the specific meaning of active stake. Staking contract `activeStake` documentation is updated now.